test/helpers/bibucketNock.ts

```diff
@@ -0,0 +1,732 @@
+          },
+        },
+        type: 'issue',
+        id: 3086,
+        repository: {
+          links: {
+            self: {
+              href: 'https://api.bitbucket.org/2.0/repositories/pypy/pypy',
+            },
+            html: {
+              href: 'https://bitbucket.org/pypy/pypy',
+            },
+            avatar: {
+              href: 'https://bytebucket.org/ravatar/%7B54220cd1-b139-4188-9455-1e13e663f1ac%7D?ts=105930',
+            },
+          },
+          type: 'repository',
+          name: 'pypy',
+          full_name: 'pypy/pypy',
+          uuid: '{54220cd1-b139-4188-9455-1e13e663f1ac}',
+        },
+        title: 'arm64 JIT: Lots of crashes',
+      },
+      content: {
+        raw: undefined,
+        markup: 'markdown',
+        html: '',
+      },
+      created_on: '2019-10-07T06:12:23.627201+00:00',
+      user: {
+        display_name: 'Stefano Rivera',
+        uuid: '{dfa073eb-c602-4740-83ef-a8fe8ee03dfd}',
+        links: {
+          self: {
+            href: 'https://api.bitbucket.org/2.0/users/%7Bdfa073eb-c602-4740-83ef-a8fe8ee03dfd%7D',
+          },
+          html: {
+            href: 'https://bitbucket.org/%7Bdfa073eb-c602-4740-83ef-a8fe8ee03dfd%7D/',
+          },
+          avatar: {
+            href:
+              'https://secure.gravatar.com/avatar/0b25ad2bcea703792d5a7bfc521a47ca?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FSR-5.png',
+          },
+        },
+        nickname: 'stefanor',
+        type: 'user',
+        account_id: '557058:efb9f360-846e-4950-afaa-c6a5da7e77b2',
+      },
+      updated_on: undefined,
+      type: 'issue_comment',
+      id: 54230712,
+    };
+  }
+}
+
+export class IssueComments {
+  issueComments: Bitbucket.Schema.PaginatedIssueComments;
+  constructor() {
+    this.issueComments = {
+      pagelen: 20,
+      values: [new IssueComment().issueComment],
+      page: 1,
+      size: 14,
+    };
+  }
+}
+
+export const getPullRequestsResponse = {
+  items: [
+    {
+      user: {
+        id: '{9d65d517-4898-47ac-9d2f-fd902d25d9f6}',
+        login: 'landtuna',
+        url: 'https://bitbucket.org/%7B9d65d517-4898-47ac-9d2f-fd902d25d9f6%7D/',
+      },
+      url: 'https://bitbucket.org/pypy/pypy/pull-requests/1',
+      body: 'Added a floor() ufunc to micronumpy',
+      createdAt: '2011-06-22T19:44:39.555192+00:00',
+      updatedAt: '2011-06-23T13:52:30.230741+00:00',
+      closedAt: null,
+      mergedAt: null,
+      state: 'DECLINED',
+      id: 1,
+      base: {
+        repo: {
+          url: 'https://bitbucket.org/pypy/pypy',
+          name: 'pypy',
+          id: '{54220cd1-b139-4188-9455-1e13e663f1ac}',
+          owner: { id: undefined, login: 'pypy', url: 'www.bitbucket.org/pypy' },
+        },
+      },
+    },
+  ],
+  totalCount: 1,
+  hasNextPage: true,
+  hasPreviousPage: false,
+  page: 1,
+  perPage: 1,
+};
+
+export const getPullRequestResponse = {
```

Comment:
Type is missing.

Reply:
Is this const necessary sice we have these clases above?

Reply:
It is a real service response? It should be moved to `__MOCKS__` folder as the Github responses are.